### PR TITLE
check that the miner is still registered every 10 minutes

### PR DESCRIPTION
**Pull request by cxmplex**
Don't do anything if its not registered once the miner is already running besides letting the user know (so we just default to the standard check_registration function), because some people like myself run registration bots that will reregister them after dereg, and I expect the miner to just pick up when it detects its registered (current behavior) again. The code also states the miner should not shut down after its been started unless explicitly caused by the user or an auto update, so this fits that as well. The only time the miner will close is at startup if not registered (current behavior).

to summarize, no behavior changes, only logging changes w/ a periodic registration check.

### CODE REVIEW

**Review by HudsonGraeme — APPROVED** [review]
Looks awesome, thank you @cxmplex!
